testsuite/replay-verify/main.py

```diff
@@ -94,13 +94,13 @@ def get_env_var(name: str, default_value: str = "") -> str:
 class ReplayConfig:
     def __init__(self, network: Network) -> None:
         if network == Network.TESTNET:
-            self.concurrent_replayer = 20
+            self.concurrent_replayer = 35
```

Comment:
This is going to OOM. We have a memory usage dash here: https://grafana.aptoslabs.com/d/b26d5174-38e7-46fe-89ac-119b4348e64d/replay-verify-on-archive?orgId=1&from=now-6h&to=now&timezone=browser&var-query0=&editIndex=0&var-Datasource=fHo-R604z&showCategory=Thresholds

Reply:
@ibalajiarun I've tried it on Testnet and Mainnet and it never triggered an OOM so I'm fairly confident this will work.